contracts/Vault.sol

```diff
@@ -722,7 +722,7 @@
         uint256 amount;
         uint256 idsLen = _ids.length;
 
-        for (uint256 i = 0; i < idsLen; ++i) {
+        for (uint256 i; i < idsLen; ++i) {
```

Comment:
This is not necessary because it's in a for loop. It only saves (a marginal amount of) gas if outside loops, sorry! Please initialize them again to shut slither up :D

Reply:
[reverted](https://github.com/lindy-labs/sc_solidity-contracts/pull/186/commits/c2588dfef87cf7401948276bb717f393c1f11a7c) :slightly_smiling_face: